src/config/Config.zig

```diff
@@ -4254,6 +4254,59 @@ pub const RepeatablePath = struct {
             // If it isn't absolute, we need to make it absolute relative
             // to the base.
             var buf: [std.fs.max_path_bytes]u8 = undefined;
+
+            // Check if the path starts with a tilde and expand it to the home directory on linux/mac
+            if (path[0] == '~') {
+                const home_env_var = switch (builtin.os.tag) {
+                    .linux, .macos => std.posix.getenv("HOME"),
+                    .windows => null,
+                    else => null,
+                };
```

Comment:
We should use `home` (from `os/homedir`) for this. It provides a more complete implementation.

I think this new code could also be generalized as an `expandHome` or similar function in that module, which would make it easier to reuse and unit test.

Reply:
Thanks, didn't know that function was there. I updated the implementation to use it and moved the code into `os/homedir` as `expandHomeUnix()`